multimedia/ffmpeg7/files/patch-libavdevice-avfoundation.diff

```diff
@@ -0,0 +1,14 @@
+--- libavdevice/avfoundation.m
++++ libavdevice/avfoundation.m
+@@ -763,6 +763,11 @@ static int get_audio_config(AVFormatContext *s)
+     return 0;
+ }
+
++#if (defined(TARGET_OS_IPHONE) && TARGET_OS_IPHONE && __IPHONE_OS_VERSION_MAX_ALLOWED < 110000) || \
++    ((!defined(TARGET_OS_OSX) || TARGET_OS_OSX) && MAC_OS_X_VERSION_MAX_ALLOWED < 101300)
```

Comment:
Just to be sure, is this condition correct? Either not defined or true?

Reply:
Yes. It is only undefined in old Mac OS X SDKs, so if we are not on OS X, it will be defined and explicitly set to false.